src/Tizen.NUI.Components/Controls/Navigation/Navigator.cs

```diff
@@ -426,7 +426,7 @@ public void Remove(Page page)
         /// </summary>
         /// <param name="index">The index of Navigator where a page will be removed.</param>
         /// <exception cref="ArgumentOutOfRangeException">Thrown when the index is less than 0, or greater than or equal to the number of pages.</exception>
-        [EditorBrowsable(EditorBrowsableState.Never)]
+        /// <since_tizen> 9 </since_tizen>
```

Comment:
if there isn't the page which has the index?

Reply:
Pushed pages are added to NavigationPages.
RemoveAt() checks if the index is valid by comparing 0 and NavigationPages.Count.
If the given index is not valid, then exception occurs.